.github/workflows/main.yml

```diff
@@ -14,14 +14,18 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-13, macos-14, linux-gpu-cuda]
-        offload: [acc, ompgpu]
+        offload: [cpu, acc, ompgpu]
         exclude:
-          - os: ubuntu-latest
-            offload: acc
+          - os: linux-gpu-cuda
+            offload: cpu
```

Comment:
gpu cuda is gets cpu, and therefore NOGPU=1?

Reply:
The idea is to not waste the GPU-enabled runner to test GPU-only code.
We already test that on the (mainstream) cpu-only runner.